config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-x-on-y.yaml

```diff
@@ -188,7 +188,7 @@ periodics:
     path_alias: k8s.io/kubeadm
   spec:
     containers:
-    - image: gcr.io/k8s-testimages/kubekins-e2e:v20201031-122dc79-1.16
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20201201-1941b8f-1.16
```

Comment:
This image doesn't exist.

Reply:
i guess this is another reason to use `latest-x.yy` everywhere, but since kubekins is ~2GB that's a lot of traffic for all the jobs as they need "Always" as the pull policy.

we can try to keep track of this image tag discrepancy manually for the time being.

Reply:
updated.